### ISSUE TITLE
Dịch ảnh Chương 4 - Phần 1

### ISSUE THREAD

**Issue by rootonchair** [aivivn/d2l-vn#835]
Các hình cần dịch:
- [ ] [singlelayer.svg](https://github.com/aivivn/d2l-vn/blob/master/img/singlelayer.svg)
- [ ] [mlp.svg](https://github.com/aivivn/d2l-vn/blob/master/img/mlp.svg)
- [ ] [capacity_vs_error.svg](https://github.com/aivivn/d2l-vn/blob/master/img/capacity_vs_error.svg)

Xem hướng dẫn dịch hình ảnh tại [đây](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)

**Comment by rootonchair** [comment]
@sudohainguyen nếu được bạn giúp mình phần này nhé

**Comment by sudohainguyen** [comment]
oke mình nhận phần này :+1:

**Comment by rootonchair** [comment]
Thank bạn

**Comment by sudohainguyen** [comment]
**singlelayer.svg**
|Tiếng Anh|Tiếng Việt|
| :---:   | :---:    |
| Input layer | Lớp đầu vào |
| Output layer | Lớp đầu ra |
---
**mlp.svg**
|Tiếng Anh|Tiếng Việt|
| :---:   | :---:    |
| Input layer | Lớp đầu vào |
| Output layer | Lớp đầu ra |
| Hidden layer | Lớp ẩn |
---
**capacity_vs_error.svg**
|Tiếng Anh|Tiếng Việt|
| :---:   | :---:    |
| Optinum | Điểm tối ưu |
| Generalization loss | Mất mát khái quát |
| Training loss | Mất mát huấn luyện |
| Model complexity | Độ phức tạp mô hình |
| Loss | Mất mát |
| Underfitting | Dưới khớp |
| Overfitting | Quá khớp |

**Comment by rootonchair** [comment]
`Generalization loss` mình chưa biết nên dịch thế nào. Mình thiên về hướng `Mất mát khái quát hóa`

**Comment by rootonchair** [comment]
@sudohainguyen Mình có để ý có `generalization error` trong [glossary](https://github.com/aivivn/d2l-vn/blob/master/glossary.md#g) là `lỗi khái quát`. Mình nghĩ có thể dịch là `mất mát khái quát` để đồng bộ

**Comment by duythanhvn** [comment]
Các chàng trai tranh thủ chốt phần này thôi nghen.
